spec/data-model/README.md

```diff
@@ -180,34 +180,31 @@ interface LiteralExpression {
   type: "expression";
   arg: Literal;
   annotation?: FunctionAnnotation | UnsupportedAnnotation;
-  attributes: Attribute[];
+  attributes: Attributes;
 }
 
 interface VariableExpression {
   type: "expression";
   arg: VariableRef;
   annotation?: FunctionAnnotation | UnsupportedAnnotation;
-  attributes: Attribute[];
+  attributes: Attributes;
 }
 
 interface FunctionExpression {
   type: "expression";
   arg?: never;
   annotation: FunctionAnnotation;
-  attributes: Attribute[];
+  attributes: Attributes;
 }
 
 interface UnsupportedExpression {
   type: "expression";
   arg?: never;
   annotation: UnsupportedAnnotation;
-  attributes: Attribute[];
+  attributes: Attributes;
 }
 
-interface Attribute {
-  name: string;
-  value?: Literal | VariableRef;
-}
+type Attributes = Map<string, Literal | VariableRef | true>;
```

Comment:
This changes the definition of attribute by adding `| true`. The syntax doesn't currently allow this?

Reply:
The `true` is meant to be used for representing value-less attributes like `@foo`, which the syntax does currently support.